sklearn/cluster/_mean_shift.py

```diff
@@ -288,7 +288,7 @@ class MeanShift(ClusterMixin, BaseEstimator):
     Parameters
     ----------
     bandwidth : float, default=None
-        Bandwidth used in the RBF kernel.
+        Bandwidth used in the flat kernel.
```

Comment:
It seems correct because we used the L2 distance that does not provide the exponential term of the RBF kernel.